mergify_engine/worker.py

```diff
@@ -551,35 +560,38 @@ class Worker:
     _worker_tasks: List = dataclasses.field(init=False, default_factory=list)
     _stream_monitoring_task: Any = dataclasses.field(init=False, default=None)
 
-    async def stream_worker_task(self, worker_id):
+    @property
+    def worker_count(self):
+        return self.worker_per_process * self.process_count
```

Comment:
Sounds that it might be a property of the `StreamSelector` itself, with `process_count` and `worker_per_process`?
Not sure, just suggesting in case it makes sense.

Reply:
It's also used by the monitoring task.